packages/cli/src/telemetry/exporter.js

```diff
@@ -0,0 +1,71 @@
+import path from 'path'
+
+import fs from 'fs-extra'
+
+import { getPaths } from '@redwoodjs/project-config'
+
+/**
+ * Custom exporter which writes spans to a file inside of .redwood/spans
+ */
+export class CustomFileExporter {
+  /**
+   * @type string
+   * @private
```

Comment:
Just curious what the `@private` directive adds if this is already marked private via the `#`?

Reply:
Pretty sure that was just auto-generated when I started typing `/**` and hit enter. I didn't think it was objectionable at the time as an additional field on the JSDoc. I don't think it's being misused here?

Reply:
I'm just asking cause I don't understand it / wanted to know if you did it for a reason. If I see something that I feel like is duplication, I'll comment on it; I'll read up on it and see what it looks like in intellisense and see if it makes a difference